tests/test_external_account_authorized_user.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+import json
+
+import mock
+import pytest  # type: ignore
+from six.moves import http_client
+
+from google.auth import exceptions
+from google.auth import external_account_authorized_user
+from google.auth import transport
+
+
+class TestCredentials(object):
```

Comment:
add tests for from_file, info, with_quota_project and anything else missing

Reply:
Done